src/olympia/devhub/templates/devhub/addons/submit/wizard.html

```diff
@@ -11,7 +11,19 @@ <h3>{{ _('Theme generator') }}</h3>
   <div id="theme-wizard" data-version="{{ version_number }}">
     <div>
         <h3>{{ _('Theme name') }}<span class="req" title="{{ _('required') }}">*</span></h3>
-        <input type="text" id="theme-name"/>
+        {% if addon %}
+          <input type="hidden" id="theme-name" value="{{ addon.name }}"/>
+          {{ addon.name }}
+          <ul class="note">
+            <li>{% trans edit_url_open=('<a href="' + url('devhub.addons.edit', addon.slug) + '">')|safe,
```

Comment:
You may want to use `edit_url_open='<a href="{}">|format_html(url('devhub.addons.edit', addon.slug'))` here. I think this can be easier on the eyes and easier to follow but also escapes the url passed in as the argument. `addon.slug` should be very safe but it's by definition user-input so needs escaping :)

Reply:
https://github.com/mozilla/addons-server/pull/8798/commits/82080c7bdaf441168218db9dc263046472be4e58